product_extended_segmentation/view/view.xml

```diff
@@ -1,5 +1,15 @@
-<?xml version="1.0"?>
+<?xml version="1.0" encoding="UTF-8"?>
 <openerp>
     <data>
+        <record id="view_compute_price_wizard" model="ir.ui.view" >
+            <field name="name">Compute Price as Sum of BoM Components and Routing</field>
+            <field name="model">wizard.price</field>
+            <field name="inherit_id" ref="product_extended.view_compute_price_wizard" />
+            <field name="arch" type="xml">
+                <xpath expr="//field[@name='recursive']" position="after">
+                    <field name="update_avg_costs"/>
```

Comment:
Add attributes so that it can be only selected if recursive is True, 
by default is False.